gsoc/project_NOvA.md

```diff
@@ -0,0 +1,7 @@
+---
+project: NOvA
+layout: default
+description: |  [NOvA](https://www-nova.fnal.gov) is a neutrino oscillation experiment whose measurement of this particle's properties relies on efficient identification and robust reconstruction of events. NOvA has already shown the first use of CNNs (convolutional neural networks) for [event classification] (http://inspirehep.net/record/1444342) in a HEP result. Now we are expanding the program of machine learning applications on NOvA events for their use in full event reconstruction and eventually event simulation applications.
```

Comment:
Unfortunately, this is still incorrect... You should keep `[NOvA]....` on a separate line. Look at `gsoc/project_SixTrack.md` for an example.